config.go

```diff
@@ -80,68 +68,76 @@ func NewDialer(cfg *Config) *uacp.Dialer {
 
 // ApplyConfig applies the config options to the default configuration.
 // todo(fs): Can we find a better name?
-//
-// Note: Starting with v0.5 this function will return an error.
-func ApplyConfig(opts ...Option) *Config {
+func ApplyConfig(opts ...Option) (*Config, error) {
 	cfg := &Config{
 		sechan:  DefaultClientConfig(),
 		session: DefaultSessionConfig(),
 	}
+	var foundErrors []error
```

Comment:
s/foundErrors/errs/

Reply:
Tried to fix this via codespaces (which is actually quite nice) but I couldn't push to your PR. Can you fix this so that we can merge?

Reply:
hmm, ok don't know how that is supposed to work with codespaces. I set it to be allowed to be edited by maintainers so not sure what else I need to set. Usually I also use the suggestion feature right besides the heading selector: 
![image](https://github.com/gopcua/opcua/assets/1769155/4ea38df7-2a32-4a34-9b3a-29f5e21b6650)
Fixed that in the latest push.